### DISCUSSION TITLE
Can't start with docker

### DISCUSSION THREAD

**Discussion by abelvieira-guisoft** [valkyrie-fnd/valkyrie#124] [Q&A]
I'm try install valkyrie in my docker, but no lucky:

[/opt/valkyrie #] docker run -v /opt/valkyrie/config.yml:/app/config.yml valkyriefnd/valkyrie -config config.yml
{"level":"error","error":"read config.yml: is a directory","time":"2023-04-06T10:52:50Z","message":"Failed to read config"}

Please can help?

**Comment by gustavste** [comment]
Hi @abelvieira-guisoft ,
Do you still have this problem or did you solve it? 
I'd make sure `/opt/valkyrie/config.yml`-file exist and looks fine.
